docs/intro.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+Polder Knowledge EntityService is a service layer to interact with any data storage you might want to use.
+By using this extra abstraction layer you will be able to create a loos binding with your data storage.
+In many situations an ORM is used to connect with a database for example. But what if your users are moved to a
+micro-service? You will have to adapt your full application? This is where this library comes in.
+
+It gives you the option to interact with any storage like it was a database. Extra functionality like sending 
+an email when a user was created can be added by attaching a listener to the required service.
+
+## Usage
+
+The example below shows a very basic setup of a service.
+
+```php
+<?php
+  //Container is your applications InteroperableContainer    
+  $entityManager = $container->get(Doctrine\ORM\EntityManager::class);
+  //Repository can be the provided ORMRepository or any other custom repository you want to use.
+  $repository = new \PolderKnowledge\EntityService\Repository\Doctrine\ORMRepository($entityManager, MyEntity::class);
```

Comment:
Just wondering if we should remove the Doctrine part of the namespace :) Maybe the class should have been named "DoctrineORM". The "Repository" part is unneeded since it's in a separate namespace already and the "Doctrine" namespace might be a bit overkill, will this result in each repository type having its own namespace?

Just a brainfart, not related to these changes though.